tsconfig.json

```diff
@@ -24,6 +24,10 @@
     "types": ["react", "node", "jest"],
     "newLine": "LF",
     "pretty": true,
+    "typeRoots": ["./@types"],
```

Comment:
**question** Do we need the `@` prefix?

Reply:
Removed the `@types` and used `types` instead. We can always switch back if this creates confusion.